app/client/ui/DocumentSettings.ts

```diff
@@ -85,6 +96,26 @@ export class DocSettingsPage extends Disposable {
               {defaultCurrencyLabel: t("Local currency ({{currency}})", {currency: getCurrency(l)})})
           )
         }),
+        dom.create(AdminSectionItem, {
+          id: 'templateMode',
+          name: withInfoTooltip(
```

Comment:
Can you move this tooltip over the `Change document type`?

I saw in other comments that we want to remove it? In that case everything looks good.

Reply:
Yes It's now removed :)